package.json

```diff
@@ -108,13 +109,13 @@
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "wp-scripts lint-style",
+    "lint:css": "wp-scripts lint-style $(git diff develop --diff-filter=d --name-only | grep -E 'css$')",
     "lint:css:fix": "npm run lint:css -- --fix",
-    "lint:js": "wp-scripts lint-js",
+    "lint:js": "wp-scripts lint-js $(git diff develop --diff-filter=d --name-only | grep -E '\\.js$')",
     "lint:js:fix": "npm run lint:js -- --fix",
-    "lint:php": "vendor/bin/phpcs",
+    "lint:php": "vendor/bin/phpcs $(git diff develop --diff-filter=d --name-only | grep -E '\\.php$')",
```

Comment:
Maybe `grep -E '\\.php$'` is overkill as opposed to `grep php`

But `grep -E '\\.js$'` is probably better than simply `grep js`